tests/torch/test_sanity_sample.py

```diff
@@ -211,7 +211,7 @@ def test_pretrained_model_eval(config, tmp_path, multiprocessing_distributed, ca
         args["--multiprocessing-distributed"] = True
 
     runner = Command(create_command_line(args, config["sample_type"]), env=ROOT_PYTHONPATH_ENV)
-    runner.run()
+    runner.retries_run()  # WA for 119128
```

Comment:
This will make any kind of failure trigger a retry. Redesign to retry on segmentation faults specifically.